src/ProfessionalGraveyard.tsx

```diff
@@ -7,42 +7,82 @@ import { shortenAddress } from './Utilities';
 
 import Burned from './img/statistics-icons/Burned.png';
 
+import Filter from './Filter';
+
+import { Trait, Attribute } from './Types';
+
+import { Traits } from './Filter';
+
 export interface GraveyardProps {
     burntSlugs: BurntSlug[];
+    traitNameMap?: Map<string, Trait>;
+    attributes?: Attribute[];
 }
 
 export function ProfessionalGraveyard(props: GraveyardProps) {
     const {
         burntSlugs,
+        traitNameMap,
+        attributes,
     } = props;
 
-    const data = React.useMemo(() => {
+    const [showFilter, setShowFilter] = React.useState<boolean>();
+
+    const handleToggleFilter = () => {
+      setShowFilter((val) => !val);
+      }
+
+      const [dataFromFilter, setDataFromFilter] = React.useState<Traits>();
+
+      if (!dataFromFilter) {
+            setDataFromFilter({
+                Background: "",
+                Slug: "",
+                Chest: "",
+                Mouth: "",
+                Head: "",
+                Eyes: "",
+                Tail:"",
+                Back: "",
+                Hands: "",
+        } as Traits)
+    }
+
+    const handleDataFromFilter = React.useCallback((data: Traits) => {
+        setDataFromFilter(data);
+    }, []);
+
+    const filteredSlugs = React.useMemo(() => {
+        if (!dataFromFilter) return burntSlugs;
```

Comment:
```suggestion
        if (!dataFromFilter) {
            return burntSlugs;
        }
```

single line returns without braces are legal but i don't like them